v20.1/time.md

```diff
@@ -1,39 +1,68 @@
 ---
-title: TIME
+title: TIME / TIMETZ
 summary: The TIME data type stores a time of day in UTC.
 toc: true
 ---
+
 The `TIME` [data type](data-types.html) stores the time of day in UTC.
 
+The `TIMETZ` data type stores a time of day with an optional time zone offset from UTC.
+
 {% include {{page.version.version}}/sql/vectorized-support.md %}
 
+## Variants
+
+`TIME` has two variants:
+
+- `TIME`, which presents all `TIME` values in UTC.
+
+- `TIMETZ`, which converts `TIME` values with an optional time zone offset from UTC.
+
+    Ordering for `TIMETZ` is done in terms of [epoch](https://en.wikipedia.org/wiki/Epoch_(computing)). Time zones with lesser values are ranked higher if times are equal. For example, `'2:00 -1' > '2:00 +0'` and `'12:00-1' > '1:00+0'`.
```

Comment:
nit: one set of examples have spaces (`'2:00 -1' > '2:00 +0'`), one doesn't ('12:00-1' > '1:00+0'`).